### PR TITLE
Count eggs

### PR DESCRIPTION
**Pull request by DanielJSottile**

### CODE REVIEW

File: scripts/sv/eggs.py [hunk 1]
```diff
@@ -32,14 +32,27 @@ def main() -> int:
...
+        nonlocal egg_count
+        egg_count += 1
+        print(f'DEBUG: You have {egg_count} eggs currently')
+
+    def restart_eggs(frame: object) -> bool:
         return time.monotonic() > start_time + 30 * 60
 
+    def reset_egg_count(vid: object, ser: object) -> None:
+        nonlocal egg_count
+        egg_count = 0
```
Comment (asottile) [inline scripts/sv/eggs.py]: I don't think this function is needed -- the program is exiting so when it starts the next time it'll already be 0

Comment (DanielJSottile) [reply scripts/sv/eggs.py]: ah that's true.  I have this for my script, which always runs.  will remove

File: scripts/sv/eggs.py [hunk 2]
```diff
@@ -160,8 +179,16 @@ def bye(vid: object, ser: object) -> None:
         'WAIT': (
             (
                 are_we_done,
-                do(Press('Y'), Wait(.5), Press('A'), Wait(5), bye),
-                'EXIT',  # TODO: loop?
+                do(
+                    Press('Y'), Wait(.5), Press('A'), Wait(5), bye,
+                ),
```
Comment (asottile) [inline scripts/sv/eggs.py]: the newlines shouldn't change here -- also the wait is different below -- it should probably match

other than that looks good

**Comment by asottile** [comment]
I think you forgot the `--boxes` argument too

**Comment by DanielJSottile** [comment]
> I think you forgot the `--boxes` argument too

yes I did.  To be fair this was marked draft and WIP :P

**Comment by asottile** [comment]
do you want to do the invalid basket loop thing in this PR too or in a separate one?

**Comment by DanielJSottile** [comment]
> do you want to do the invalid basket loop thing in this PR too or in a separate one?

Yeah sure.  I added it earlier here.

**Review by asottile — APPROVED** [review]

**Pull request merged**